tests/esm-path-mapping/index.ts

```diff
@@ -0,0 +1,32 @@
+import * as assert from 'assert';
```

Comment:
We can made a CJS version of `esm-path-mapping` at `cjs-path-mapping` to test that this also works for CJS.